README.md

```diff
@@ -184,20 +184,25 @@ Submits a map of custom `(key, value)` pairs to be included in the data for the
 Method accepts a single parameter, a JavaScript object with values of type `String`.
 `key` is limited to 80 characters and must be unique; `value` is limited to 1000 characters.
 
-#### areCookiesEnabled() => boolean
-Utility method to determine if cookies are enabled on the client's browser.  If cookies are not enabled, the
-quiq-chat library will not function properly.
+#### isStorageEnabled = () => boolean
+Utility function to tell the client if quiq-chat has the capability to set its required data.
+Note this will return true if quiq-chat is only able to set its data in a non-peristent way.
```

Comment:
Wait, yep that would mean it would return true if it's also able to set it in a persistent way... ok, will update doc